From b6eb208b29ae720e45a2950453fa4278a88bbcc9 Mon Sep 17 00:00:00 2001
From: Kai-Heng Feng <kai.heng.feng@canonical.com>
Date: Tue, 16 Jun 2020 13:24:27 +0800
Subject: [PATCH] hwdb: Add EliteBook to use micmute hotkey
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/systemd/+bug/1890448
Origin: upstream, https://github.com/systemd/systemd/commit/b6eb208b29ae720e45a2950453fa4278a88bbcc9

Like HP ZBooks, all EliteBooks use the same micmute scancode.
---
 hwdb.d/60-keyboard.hwdb | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -584,6 +584,9 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett
 
 # HP EliteBook 725 G2
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pnHPLicrice:pvr*
+# HP EliteBook
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pnHPEliteBook*:pvr*
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHP*:pnHPEliteBook*:pvr*
 # HP ProBook 440 G2
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pnHP440G2:pvr*
 # several HP ProBooks 4xx
@@ -610,7 +613,6 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHP:pnHP
 
 # HP Folio 1040g2
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pnHPEliteBookFolio1040G2:pvr*
- KEYBOARD_KEY_81=f20                                    # Fn+F8; Microphone mute button, should be micmute
  KEYBOARD_KEY_d8=!f23                                   # touchpad off
  KEYBOARD_KEY_d9=!f22                                   # touchpad on
 
